rpcs3/Emu/RSX/rsx_cache.h

```diff
@@ -199,14 +199,17 @@ namespace rsx
 		{
 			if (!g_cfg.video.disable_on_disk_shader_cache)
 			{
-				root_path = rpcs3::cache::get_ppu_cache() + "shaders_cache";
+				if (std::string cache_path = rpcs3::cache::get_ppu_cache(); !cache_path.empty())
+				{
+					root_path = std::move(cache_path) + "shaders_cache/";
+				}
```

Comment:
It's probably better to throw an exception.
How is this even a use-case ?

Reply:
It is and I'm planning to use it.

Reply:
how

Reply:
Savestates debugging mode without requiring game files so no cache.

Reply:
then maybe add a fatal or an error here.

Reply:
It's logged in rpcs3::cache::get_ppu_cache() already.

Reply:
but only as a warning.
regular users won't see that

Reply:
Fixed.